architecture/2024-08-applications-rp-component-threat-model.md

```diff
@@ -0,0 +1,206 @@
+# Radius Applications RP Component Threat Model
+
+- **Author**: @nithyatsu
+
+## Overview
+
+This document provides a threat model for the Radius Applications RP component. It identifies potential security threats to this critical part of Radius and suggests possible mitigations. The document includes an analysis of the system, its assets, identified threats, and recommended security measures to protect the system.
+
+The Applications RP component is responsible for managing applications and their resources. It communicates with UCP, Deployment Engine and Controller components to achieve this. 
```

Comment:
Does the Controller directly talk to the Applications-RP?

Reply:
I think we could take this paragraph out. It duplicates content that's in the system description.